dds_web/__init__.py

```diff
@@ -282,13 +282,15 @@ def load_user(user_id):
             send_usage,
             collect_stats,
             monitor_usage,
-            update_unit,
+            update_unit_sto4,
```

Comment:
why do we need to be specific about the unit's s3 storage? 
If/when we change s3 provider, or if we use a particular s3 provider for a particular unit, we will need to change command name again or/and add a new command

Reply:
I chose that name because the command specific sets the sto4 fields in the DB, and it has harcoded "sto4" as a string as well, so I figured it made sense as it is right now

Reply:
We created this command when we were migrating from sto2 to sto4 and I believe we had the intention from the beginning on using `update_unit` as a command that could be used to update other things (and not just s3 info) in the future. We have the different options as `xxx_sto4` to match the fields in the database as @rv0lt said. I think for now we can keep it like this, and potentially change it when we remove the `sto2` info from the database in coming sprints.

Reply:
we agreed to take this in a separate task/PR

Reply:
Okay, just one thought, tho. 

If the intention was for this command to be more general, then it needs some refactoring, that's why I opted for a new command, it seemed cleaner as it is right now